### DISCUSSION TITLE
How do you prevent the nvim-tree split pane from becoming a buffer when the cursor is inside?

### DISCUSSION THREAD

**Discussion by niamleeson** [nvim-tree/nvim-tree.lua#2765] [Q&A]
I often accidentally leave the cursor inside the nvim-tree split pane and switch to a different buffer and the pane ends up becoming that buffer. How do I prevent this from happening?

**Comment by alex-courtis** [comment]
That's not intended default behaviour, see `:help nvim-tree.actions.open_file.eject`

Please raise a [Bug Report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with minimal configuration so that we may reproduce and resolve this issue.

**Reply by niamleeson** [reply]
Thank you! The configuration worked!
